springlab/index.html

```diff
@@ -314,5 +314,5 @@ <h2>
 		</div>
 	</footer>
 </body>
-
+<script type="text/javascript" src="../assets/fathom.js></script>
```

Comment:
Change script file to be absolute, and add missing `"` to the `src` tag.

```suggestion
<script type="text/javascript" src="/assets/fathom.js"></script>
```